tools/functional/api/test_merge_requests.py

```diff
@@ -29,6 +29,16 @@ def test_merge_requests(project):
     )
 
 
+def test_merge_request_get(project):
```

Comment:
What special case is tested here? Maybe we can get rid of that comment by having a more elaborate test name?

```suggestion
def test_merge_request_get_lazy_returns_mr_iid(project):
```

Reply:
> What special case is tested here?

No special case, just testing that mergerequest.get() works for both a standard get() and get(..., lazy=True).

I would like to put them both in the same function as it takes less time as no need to setup/tear-down with a second function.

Does that answer your question?

Thanks.

Reply:
The time doesn't really matter. Code quality matters a bit more imo.

Reply:
to move this along, @JohnVillalovos since this PR was originally opened you have added some more specific tests here https://github.com/python-gitlab/python-gitlab/blob/master/tests/functional/api/test_merge_requests.py#L103, maybe the name could be similar in this one to reflect it's a regression test for https://github.com/python-gitlab/python-gitlab/issues/1425.